### ISSUE TITLE
batch insert but only one recode 

### ISSUE THREAD

**Issue by hejian1639** [housepower/ClickHouse-Native-JDBC#85]
I used latest version library, my code is below 

```
 Class.forName("com.github.housepower.jdbc.ClickHouseDriver");
        Connection connection = DriverManager.getConnection("jdbc:clickhouse://172.17.162.177:9000/default");

        Statement stmt = connection.createStatement();
        stmt.executeQuery("drop table if exists test_jdbc_example");
        stmt.executeQuery("create table test_jdbc_example(day Date, name String, age UInt8, data Array(UInt32)) Engine=Log");

        PreparedStatement pstmt = connection.prepareStatement("INSERT INTO test_jdbc_example VALUES(?, ?, ?, ?)");

        Object[] objects = new Object[]{
                1, 3, 9
        };

        for (int i = 0; i < 200; i++) {
            Array array = connection.createArrayOf("UInt32", objects);
            pstmt.setDate(1, new Date(System.currentTimeMillis()));
            pstmt.setString(2, "Zhang San" + i);
            pstmt.setByte(3, (byte) i);
            pstmt.setArray(4, array);
            pstmt.addBatch();
        }
        pstmt.executeBatch();
```

but result is below

2019-04-24	Zhang San0	0	[1,3,9]
2019-04-24	Zhang San1	1	
2019-04-24	Zhang San2	2	
2019-04-24	Zhang San3	3	
2019-04-24	Zhang San4	4	
....

so why only first line got correct data ?

thanks,
John

**Comment by sundy-li** [comment]
Sorry for the delay, this bug is fixed in ac9391585ded977d803c61a04d0ee1c06443fdce.

**Comment by tauitdnmd** [comment]
@sundy-li  could you please close this issue cause it was fixed long ago
